query/ec2/ec2_instance_no_high_level_finding_in_inspector_scan.sql

```diff
@@ -0,0 +1,16 @@
+select
```

Comment:
This check need to query `aws_inspector_finding` (where asset_type = 'ec2-instance') table and join with aws_ec2_instance to find if any instance have high level finding.